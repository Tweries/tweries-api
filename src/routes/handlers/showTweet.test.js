const showTweet = require('./showTweet');

test('showTweet', async () => {
  const mockGet = jest.fn(() => ({}));

  const options = {
    tweetUrl: 'https://twitter.com/trisweb/status/1200501818461212672',
    T: { get: mockGet }
  };

  await showTweet(options);

  expect(mockGet.mock.calls[0][1]).toMatchSnapshot();
});
