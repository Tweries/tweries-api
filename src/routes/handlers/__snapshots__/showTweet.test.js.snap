// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`showTweet 1`] = `
Object {
  "id": "1200501818461212672",
}
`;
